### PR TITLE
fix cosmos stake direction

### PR DESCRIPTION
**Pull request by Pantani**
`/v1/cosmos/cosmos1dx27g0kzhwej0ekcf2k9hsktcxnmpl7fcehcvq`

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -91,13 +91,17 @@ func makeTxRoute(router gin.IRouter, api blockatlas.Platform, path string) {
 
 		page := make(blockatlas.TxPage, 0)
 		for _, tx := range txs {
+			if tx.Direction != "" {
+				goto AddTx
```
Comment (vikmeup) [inline api/handlers.go]: is it safe to use this? maybe consider if else instead?

File: platform/cosmos/api.go [hunk 2]
```diff
@@ -158,8 +158,10 @@ func fillDelegate(tx *blockatlas.Tx, delegate MessageValueDelegate, msgType stri
 	title := blockatlas.KeyTitle("")
 	switch msgType {
 	case MsgDelegate:
+		tx.Direction = blockatlas.DirectionOutgoing
 		title = blockatlas.AnyActionDelegation
```
Comment (vikmeup) [inline platform/cosmos/api.go]: can you also add this: https://github.com/trustwallet/blockatlas/issues/617, should be a quick one

File: api/handlers.go (hunk 1 above)
Comment (Pantani) [reply api/handlers.go]: yess!!! `goto` method is normal and safe in Go, the golang compiler cannot compile the app if the `goto` skip handling some error or instantiate a variable. you can only use `goto` in safe places

File: platform/cosmos/api.go (hunk 2 above)
Comment (Pantani) [reply platform/cosmos/api.go]: this is not so easy because we need to filter 2 log messages, I going to do that in another PR. And keep this PR simple

**Pull request merged**